components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/introspection/IntrospectionResponseBuilderTest.java

```diff
@@ -167,6 +188,8 @@ public void testResponseBuilderWithAdditionalProperties() throws Exception {
         Map<String, Object> additionalData = new HashMap<>();
         List<Map<String, Object>> permissions = new ArrayList<>();
 
+        mockOAuthServerConfiguration();
```

Comment:
In addition write the unit test for the configuration, we need to assign some values for the config and assert the introspection response in order to completely test the flow.

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1708/commits/e3de4bc40dc1f52a171fbc94dbe94022023ddc23